cmd/version/version.go

```diff
@@ -0,0 +1,27 @@
+package version
+
+import (
+	"fmt"
+	"runtime/debug"
+
+	"github.com/spf13/cobra"
+)
+
+var Version string = "dev"
+
+var VersionCmd = &cobra.Command{
+	Use: "version",
```

Comment:
no need to make a whole command out of this a variable should do. check this out: https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications